test_setup/fixtures/raw_list.json

```diff
@@ -8,14 +8,12 @@
         "key": "2000835",
         "caption": "Testpreisliste Lieferanten"
       },
-      "properties": [
-        {
+      "properties": {
```

Comment:
weird, this didn't break anything ?

Reply:
It actually fixed the the unit test which was passing an array